sig-network/README.md

```diff
@@ -135,6 +148,31 @@ The following [subprojects][subproject-definition] are owned by sig-network:
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 [working-group-definition]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
 <!-- BEGIN CUSTOM CONTENT -->
+
+### Subproject Leads
+
+SIG network provides some additional responsibilities for subproject leads beyond was in covered in the [standard subproject definition][subprojects].
+Most of these additional responsibilities relate to communication about their projects within SIG Network, and to the greater community:
+
+- **Transparent Project Planning, Maintenance and Communication:**
+  - Subproject Leads **MUST** provide transparent view into their historical and future project plans (eg; using GitHub project boards, KEPs, or custom enhancement proposals (see [GEPs]/[NPEPs])).
+  - Subproject leads **MUST** create, maintain and be present in a public [Kubernetes Slack] channel with the naming `#sig-network-<subproject>`.
+  - Subproject leads **SHOULD** create and maintain a regular public [Zoom] sync on the [SIG Network Calendar].
+  - Subproject leads **MUST** keep the project in good health through regular issue triaging, PR reviews, CI health monitoring and checking testgrids (delegating this to other members in the community is good!)
```

Comment:
While it's absolutely true that we want subprojects to do this, this one is difficult.

With the above MUST items (make your process clear, and create a channel) these are easy things to identify when they aren't happening and just point to this doc. "good health" for a project however can be really subjective :thinking: I think I'm in favor of dropping this one for now, if we feel in the future there's some need related to this I think it should be its own effort.

```suggestion
```

Reply:
hmm but what's the point of being a lead if your project members are not having a good experience... so what I meant in this point with issue triage and PR reviews is that I am not necessarily saying the lead has to be the one doing it; but the lead needs to make sure "its happening somehow ;; either the members or rotating with other leads etc..."

the CI health monitoring and testgrid was @aojea 's suggestion I'll let him chime in

Reply:
In retrospect, I rescind my thought about dropping this. Let's just use the softer must language and consider this resolved.